controllers/upgrade_handlers.go

```diff
@@ -148,6 +148,11 @@ func (u *UpgHandler) PrePivot(ctx context.Context, ibu *lcav1alpha1.ImageBasedUp
 	}
 
 	u.Log.Info("Writing extra-manifests into new stateroot")
+	labels := map[string]string{"lca.openshift.io/for-ocp-version": ibu.Spec.SeedImageRef.Version}
+	if err := u.ExtraManifest.ExtractAndExportManifestFromPoliciesToDir(ctx, labels, staterootVarPath); err != nil {
```

Comment:
How do we handle the no policy case (i.e. SNO that is not using acm/gitops) ?

Reply:
Maybe another spec field where the actual manifests are stored in config maps, instead of just name/kind for exporting.

Reply:
Will add a CRD check